static/javascripts/script.js

```diff
@@ -129,6 +129,7 @@
 			'logsclient': null,
 			'logshare': null,
 			'wafanalyzer': null,
+      'json-schema-tools': null,
```

Comment:
nitpick: can you match the whitespace?

Reply:
Ah, yes, must have been an editor setting thing.